backend/controllers/BroadcastController.ts

```diff
@@ -76,11 +77,50 @@ async function updateTwilioStatus(req: Request, res: Response) {
   return AppResponse.ok(res, {}, 204)
 }
 
+async function commentChangeSubscription(req: Request, res: Response) {
+  if (
+    !req.body.conversation || !Array.isArray(req.body.conversation.external_authors) ||
+    req.body.conversation.external_authors.length === 0
+  ) {
+    return AppResponse.badRequest(res, 'Invalid or missing conversation data in request body')
+  }
+
+  const phoneNumber = req.body.conversation.external_authors[0]?.phone_number
+  if (!phoneNumber) {
+    return AppResponse.badRequest(res, 'Phone number not found in request body')
+  }
+
+  if (!req.body.comment || !req.body.comment.author || !req.body.comment.author.name) {
+    return AppResponse.badRequest(res, 'Invalid or missing author name in comment')
+  }
+  const authorName = req.body.comment.author.name
+
+  let isUnsubscribe: boolean
+  if (req.path === Paths.Comment.Unsubscribe.toString()) {
+    isUnsubscribe = true
+  } else if (req.path === Paths.Comment.Resubscribe.toString()) {
+    isUnsubscribe = false
+  } else {
+    return AppResponse.badRequest(res, 'Invalid request path')
+  }
+
+  try {
+    await BroadcastService.updateSubscriptionStatus(phoneNumber, isUnsubscribe, authorName)
+    return AppResponse.ok(res, {
+      message: `Author ${isUnsubscribe ? 'unsubscribed' : 'resubscribed'} successfully`,
+    }, 200)
+  } catch (error) {
+    console.error('Error in commentChangeSubscription:', error)
```

Comment:
```suggestion
    logger.error('Error in commentChangeSubscription:', error)
```
You should print the `phoneNumber, isUnsubscribe, authorName` out